docs/modules/ROOT/pages/cli.adoc

```diff
@@ -406,23 +406,23 @@ Execute a script in the context of NRE.
 
 ===== Arguments
 
-- _PATH_TO_SCRIPT_
+- `*PATH_TO_SCRIPT*`
 +
 Path to the script to run.
 
 ===== Options
 
-- _--network_
+- `*--network*`
 +
-Select network: one of ('localhost', 'integration', 'goerli', 'goerli2', 'mainnet').
+Select network: one of (localhost, integration, goerli, goerli2, mainnet).
```

Comment:
i'm starting to ask myself whether we should abstract this into its own subsection and link to it, instead of repeating it everywhere

Reply:
Added snippets to handle these repetitions at least on code, but not sure about moving it into another section and linking, I think if I were going to check a command I would appreciate having the information right there (no extra click for UX).

Reply:
i like your snippets solution better 👍